docs/ref/contrib/postgres/indexes.txt

```diff
@@ -9,6 +9,17 @@ PostgreSQL specific model indexes
 The following are PostgreSQL specific :doc:`indexes </ref/models/indexes>`
 available from the ``django.contrib.postgres.indexes`` module.
 
+``BrinIndex``
+=============
+
+.. class:: BrinIndex(pages_per_range=None)
+
+    Creates a `Brin index
+    <https://www.postgresql.org/docs/devel/static/brin-intro.html>`_. For
```

Comment:
Good. Will also make it consistent with the other links to PostgreSQL's documentation.